jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3ViewerPanel.java

```diff
@@ -84,23 +85,23 @@ public void actionPerformed(ActionEvent ae) {
         arnText.setComponentPopupMenu(menu);
 
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
-            s3TreeNode = new S3TreeDirectoryNode(s3client, bucketVirtual.getName(), null, "");
+            s3TreeNode = new S3TreeDirectoryNode(bucketVirtual, null, "");
 
-            ColumnInfo key = new S3Column(S3ColumnType.NAME);
-            ColumnInfo size = new S3Column(S3ColumnType.SIZE);
-            ColumnInfo modified = new S3Column(S3ColumnType.LAST_MODIFIED);
+            ColumnInfo<Object, String> key = new S3Column(S3ColumnType.NAME);
+            ColumnInfo<Object, String> size = new S3Column(S3ColumnType.SIZE);
+            ColumnInfo<Object, String> modified = new S3Column(S3ColumnType.LAST_MODIFIED);
             final ColumnInfo[] COLUMNS = new ColumnInfo[] {key, size, modified};
             model = createTreeTableModel(COLUMNS);
 
             S3TreeCellRenderer treeRenderer = new S3TreeCellRenderer();
             DefaultTableCellRenderer tableRenderer = new DefaultTableCellRenderer();
             tableRenderer.setHorizontalAlignment(SwingConstants.LEFT);
-            /**
+
+            /*
```

Comment:
this comment seems to be hanging out here not attached to anything

Reply:
right - deleted.